### ISSUE TITLE
mrmatrix: coarsen should create nan_values at each resolution?

### ISSUE THREAD

**Issue by mccalluc** [higlass/clodius#62]
@pkerpedjiev : Am I understanding it correctly, that there should be a nan_values at each resolution? It is not producing that right now. Without that, it would still need to consult the large matrix at resolution 1 to calculate averages. Assign back to me if there's work to do.

`ts_to_mrmatrix_test.py`:
```
class ParseTest(unittest.TestCase):
    def test_parse(self):
        ...
        self.assertEqual(list(hdf5['resolutions']['1'].keys()), ['nan_values', 'values'])
        ...
        self.assertEqual(list(hdf5['resolutions']['2'].keys()), ['values'])
```

**Comment by pkerpedjiev** [comment]
Yes, that's correct, there should be `nan_values` assigned at every resolution if you want to be able to view averages at lower resolutions. Enabling that is just a matter of doing the same thing that `coarsen` (https://github.com/higlass/clodius/blob/1e275f4ce4aba268d2a538bc15cde90fdd08d864/scripts/tsv_to_mrmatrix.py#L18) does except with the `nan_values` dataset.

I don't think I ever implemented it because I didn't need to get average values.
